qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -185,6 +185,7 @@ def coherence_limit(nQ=2, T1_list=None, T2_list=None, gatelen=0.1):
     @staticmethod
     def calculate_1q_epg(
         epc_1_qubit: float,
+        epc_1_qubit_stderr: float,
```

Comment:
Can we directly get EPC `FitVal` object, rather than decomposing it?

Reply:
I agree, that's better. Fixed.